Source/Schema.NET/ValuesConverter.cs

```diff
@@ -40,8 +34,147 @@ public class ValuesConverter : JsonConverter
             JsonReader reader,
             Type objectType,
             object existingValue,
-            JsonSerializer serializer) =>
-            throw new NotImplementedException();
+            JsonSerializer serializer)
+        {
+            var mainType = UnwrapNullable(objectType);
+
+            object argument = null;
+
+            var tokenType = reader.TokenType;
+            var value = SanitizeReaderValue(reader, tokenType);
+
+            JToken token = JToken.Load(reader);
+            if (mainType.GenericTypeArguments.Length == 1)
```

Comment:
There are two major blocks: Values<T> and the other Values<T....>. For this one we only need to figure the correct type and set "argument" accordingly, to be used in the end return.